kepler/tasks.py

```diff
@@ -76,12 +78,33 @@ def submit_to_dspace(job, data):
         pkg.metadata = _fgdc_to_mods(get_fgdc(data))
         with tempfile.NamedTemporaryFile(suffix='.zip') as fp:
             pkg.write(fp)
-            handle = sword.submit(current_app.config['SWORD_SERVICE_URL'], fp.name,
-                                  auth=(username, password))
+            handle = sword.submit(current_app.config['SWORD_SERVICE_URL'],
+                                  fp.name, auth=(username, password))
         job.item.handle = handle
         db.session.commit()
 
 
+def get_geotiff_url_from_dspace(job):
+    """Retrieve the GeoTIFF URL from a DSpace Handle.
+
+        .. note:: assumes the OAI-ORE only contains a single TIFF.
+
+        :param job: :class:`~kepler.models.Job`
+    """
+    handle = job.item.handle.replace('http://hdl.handle.net/', '')
+    ore_url = current_app.config['OAI_ORE_URL'] + handle + '/ore.xml'
+    r = requests.get(ore_url)
+    r.raise_for_status()
+    doc = ET.fromstring(r.text)
+    image_urls = doc.findall(
```

Comment:
I know we should only have one .tif url but I'm a little concerned about silently accepting multiples. I'd rather find out our assumption is false than having it just take the last one it finds. Since we're using lxml we can use xpath and probably just check to make sure the query only returns one item. Maybe something like:
```python
tif_urls = doc.xpath('//rdf:Description/@rdf:about[contains(., ".tif")]',
                     namespaces={'rdf': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#'})
if len(tif_urls) != 1:
    raise MiserableFailure()
else:
    job.item.geotiff_url = tif_url[0]